packages/plugin-analytic-fb-pixel/package.json

```diff
@@ -34,7 +34,7 @@
     "@ima/plugin-analytic": "^4.0.1"
   },
   "peerDependencies": {
-    "@ima/plugin-script-loader": "1.x"
+    "@ima/plugin-script-loader": "^1.x || ^2.x"
```

Comment:
There is no point in adding `^`, since we are accepting any minor version already.